docs/scripts/fledge_plugin_list

```diff
@@ -172,3 +269,36 @@ to extend the set of notification deliveries.
 
 EOF6
 table "$REPOSITORIES" "Fledge Notification Delivery Plugins" notify
+
+if [[ -d /tmp/keywords ]]; then
+	rm -rf /tmp/keywords
+fi
```

Comment:
For the last attempt?

Reply:
Yes, final cleanup to prevent leaving a lot of files around